templates/jicofo-deployment.yaml

```diff
@@ -68,7 +68,11 @@ spec:
       {{- if $.Values.jicofo.extraEnvs }}
         {{- toYaml $.Values.jicofo.extraEnvs | nindent 8 }}
       {{- end }}
+      {{- if $.Values.jicofo.image }}
```

Comment:
This value (and all the other `image` values) is defined by default, so you have to explicitly nuke them out when using to get the other branch.

I don't think there's a compatibility problem removing the defaults now that we have the other branch